dlpy/model_conversion/write_keras_model_parm.py

```diff
@@ -45,8 +46,11 @@ def write_keras_hdf5_from_file(model, hdf5_in, hdf5_out):
 
     '''
     # open input/output files
-    f_in = h5py.File(hdf5_in, 'r')
-    f_out = h5py.File(hdf5_out, 'w')
+    if os.path.isfile(hdf5_in):
+        f_in = h5py.File(hdf5_in, 'r')
+        f_out = h5py.File(hdf5_out, 'w')
```

Comment:
I'm wondering if you should check to see if the output directory exists at the least.  I'm not sure there's a good way to check to see if the output file is writable completely.

Reply:
I added a try/except block to catch problems writing an output file.  According to the H5PY docs, the File object should behave like a standard Python file object, so IOError will be raised if there are any problems opening the file in write mode.  I also added a check for the directory separator when calling the rsplit() function so that it doesn't fail if we are reading from a root directory.